docs/contributing/contributing-releases/README.md

```diff
@@ -30,6 +30,8 @@ We follow a monthly release cadence. Any contributions that have been merged thr
 
 ## Release Process
 
+For the entire release process, directly clone repositories under the radius-project organization and create branches off of them. Do not create branches in your personal forks when creating pull requests.
```

Comment:
What's the error that we run into when creating PRs from forks?

Reply:
Error was in accessing github token. https://github.com/radius-project/radius/actions/runs/8820651650/job/24214732371